streams/il.m3u

```diff
@@ -17,8 +17,8 @@ https://bcovlive-a.akamaihd.net/773a2fa387914315ad11e6957cd54f6e/eu-central-1/53
 https://live.pars-israel.com/iptv/stream.m3u8
 #EXTINF:-1 tvg-id="KabbalahforthePeopleIsrael.il",Kabbalah for the People (Israel) (504p) [Not 24/7]
 https://edge3.uk.kab.tv/live/tv66-heb-high/playlist.m3u8
-#EXTINF:-1 tvg-id="Kan11.il",KAN 11 Israel (360p) [Geo-blocked]
-https://kan11.media.kan.org.il/hls/live/2024514/2024514/master.m3u8
```

Comment:
the old link works too

Reply:
Not GEO in France + 1080p instead of 360p

Reply:
it's adaptative, the link supports 360p 720p 1080p

Reply:
> the old link works too

got it, was confused on what to do.